### PR TITLE
SIGINT-833 update task.json

### PR DESCRIPTION
**Pull request by lsynopsys**

### CODE REVIEW

File: synopsys-task/task.json [hunk 1]
```diff
@@ -81,123 +81,123 @@
...
+      "helpMarkDown": "Directory path to install Coverity"
     },
     {
       "name": "bridge_coverity_connect_policy_view",
       "type": "string",
       "label": "Coverity Connect Policy View",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Coverity Connect Policy View"
+      "helpMarkDown": "ID number of a saved view to apply as a 'break the build' policy. If any defects are found within this view when applied to the project, the build will be failed with an exit code."
```
Comment (lsynopsys) [inline synopsys-task/task.json:143]: Should we replace ID with Name?

File: synopsys-task/task.json [hunk 2]
```diff
@@ -12,23 +12,23 @@
...
-  "instanceNameFormat": "synopsystask",
+  "instanceNameFormat": "Synopsys Security Scan",
   "inputs": [
     {
       "name": "bridge_download_url",
       "type": "string",
       "label": "Bridge download url",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "A bridge_download_url"
```
Comment (sig-tithi) [inline synopsys-task/task.json:23]: URL to download Synopsys Bridge.

File: synopsys-task/task.json [hunk 3]
```diff
@@ -44,31 +44,31 @@
       "label": "Synopsys Bridge version",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Bridge version to be downloaded"
+      "helpMarkDown": "Specified version of Synopsys Bridge will be downloaded and configured"
```
Comment (sig-tithi) [inline synopsys-task/task.json:47]: Bridge version to be downloaded is fine.

File: synopsys-task/task.json [hunk 4]
```diff
@@ -81,123 +81,123 @@
...
+      "helpMarkDown": "All diagnostics files will be available to download when 'true' passed"
     },
     {
       "name": "blackduck_automation_fixpr",
       "type": "string",
       "label": "Blackduck Automation Fix PR",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Blackduck Automation Fix PR"
+      "helpMarkDown": "Flag to enable automatic creation for fix pull request when Black Duck vulnerabilities reported. By default fix pull request creation will be disabled"
```
Comment (sig-tithi) [inline synopsys-task/task.json:184]: We discussed that text suggested in Raj's PPT we will add.

**Review by sig-tithi — APPROVED** [review]

File: synopsys-task/task.json [hunk 5]
```diff
@@ -81,123 +81,131 @@
...
+      "helpMarkDown": "All diagnostics files will be available to download when 'true' passed"
     },
     {
       "name": "blackduck_automation_fixpr",
       "type": "string",
       "label": "Blackduck Automation Fix PR",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Blackduck Automation Fix PR"
+      "helpMarkDown": "Applicable to push events only"
```
Comment (kishorikumar) [inline synopsys-task/task.json:192]: I believe, we need to put "Requires azure_token" as per the ADO mocked UI.

File: synopsys-task/task.json [hunk 6]
```diff
@@ -81,123 +81,131 @@
...
     },
     {
       "name": "bridge_coverity_automation_prcomment",
       "type": "string",
       "label": "Coverity Automation PR Comment",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Coverity Automation PR Comment"
+      "helpMarkDown": "Applicable to pull requests only"
```
Comment (kishorikumar) [inline synopsys-task/task.json:200]: I believe, we need to put "Requires azure_token" as per the ADO mocked UI.

File: synopsys-task/task.json [hunk 7]
```diff
@@ -81,123 +81,131 @@
...
     },
     {
       "name": "bridge_blackduck_automation_prcomment",
       "type": "string",
       "label": "Blackduck Automation PR Comment",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Blackduck Automation PR Comment"
+      "helpMarkDown": "Applicable to pull requests only"
```
Comment (kishorikumar) [inline synopsys-task/task.json:208]: I believe, we need to put "Requires azure_token" as per the ADO mocked UI.

**Review by kishorikumar — CHANGES_REQUESTED** [review]

**Pull request merged**